platforms/evm/src/signer.ts

```diff
@@ -135,20 +140,11 @@ export class EvmNativeSigner<N extends Network, C extends EvmChains = EvmChains>
         ...gasOpts,
         from: this.address(),
         nonce: await this._signer.getNonce(),
+        // Override any existing values with those passed in the constructor
+        ...this.opts?.overrides,
       };
 
-      // try {
-      //   const estimate = await this._signer.provider!.estimateGas(t);
-      //   t.gasLimit = estimate + estimate / 10n; // Add 10% buffer
-      //   if (this.opts?.maxGasLimit && t.gasLimit > this.opts?.maxGasLimit) {
-      //     throw new Error(
-      //       `Gas limit ${t.gasLimit} exceeds maxGasLimit ${this.opts?.maxGasLimit}`,
-      //     );
-      //   }
-      // } catch (e) {
-      //   console.info('Failed to estimate gas for transaction: ', e);
-      //   console.info('Using gas limit: ', t.gasLimit);
-      // }
+      console.log(t);
```

Comment:
What's the stance on console logging in the SDK?

Reply:
ah good catch, should yoink this one

Generally though, avoid `log` unless some debug setting is avail/set